### PR TITLE
play 3.0 upgrade

### PR DESCRIPTION
**Pull request by Michael-RJ**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -44,44 +43,32 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
-  "org.jsoup"              %  "jsoup"                  % "1.15.3"             % scope,
-  "org.scalatestplus"      %% "mockito-3-3"            % "3.2.2.0"            % scope
-)
-
 def oneForkedJvmPerTest(tests: Seq[TestDefinition]): Seq[Group] = tests map {
   test => Group(test.name, Seq(test), SubProcess(
     ForkOptions().withRunJVMOptions(Vector("-Dtest.name=" + test.name))))
 }
 
+ThisBuild / scalaVersion := "2.13.12"
```
Comment (derickdamoah) [inline build.sbt:51]: scala version can go up to 2.13.14

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  val it: Seq[ModuleID] = Seq()
+
+}
+
+object CompileDependencies {
+  val bootstrapVersion = "8.5.0"
+
+  val compile: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"       %% "bootstrap-backend-play-30" % bootstrapVersion,
+    "com.typesafe.play" %% "play-json-joda"            % "2.10.0-RC7"
```
Comment (derickdamoah) [inline project/AppDependencies.scala:32]: play-json-joda should be "org.playframework" %% "play-json-joda" % "3.0.3"

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  )
+
+  def apply(): Seq[ModuleID] = compile
+}
+
+object UnitTestDependencies extends CommonTestDependencies {
+  override val scope: Configuration = Test
+
+  override val test: Seq[ModuleID] = Seq(
+    "org.jsoup"         %  "jsoup"                    % "1.15.3"             % scope,
```
Comment (derickdamoah) [inline project/AppDependencies.scala]: jsoup can go up to 1.172

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    "org.scalatestplus" %% "mockito-3-4"              % "3.2.10.0"           % scope,
```
Comment (derickdamoah) [inline project/AppDependencies.scala]: mockito has moved to "org.scalatestplus" %% "mockito-4-2" % "3.2.11.0" % Test

File: project/build.properties [hunk 5]
```diff
@@ -1 +1 @@
-sbt.version=1.7.2
+sbt.version=1.9.7
```
Comment (derickdamoah) [inline project/build.properties:1]: sbt can go higher 1.9.9

File: project/plugins.sbt [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.21.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.2.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.5.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.19")
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.2")
 
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.9.3")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "2.0.11")
```
Comment (derickdamoah) [inline project/plugins.sbt:13]: sbt scoverage can go up to 2.0.12

**Review by derickdamoah — COMMENTED** [review]
Can change the readme to add instructions on how to run coverage tests?

File: project/AppDependencies.scala (hunk 4 above)
Comment (Michael-RJ) [reply project/AppDependencies.scala]: Done.

File: project/AppDependencies.scala (hunk 3 above)
Comment (Michael-RJ) [reply project/AppDependencies.scala]: Done.

File: project/build.properties (hunk 5 above)
Comment (Michael-RJ) [reply project/build.properties:1]: @derickdamoah I tend to align to latest versions shown within tax catalogue even if not latest version.

File: build.sbt (hunk 1 above)
Comment (Michael-RJ) [reply build.sbt:51]: @derickdamoah I tend to align to latest versions shown within tax catalogue even if not latest version.

File: project/AppDependencies.scala (hunk 2 above)
Comment (Michael-RJ) [reply project/AppDependencies.scala:32]: @derickdamoah I understand Joda library should now be migrated to use Java alternative.

**Review by derickdamoah — DISMISSED** [review]

**Review by derickdamoah — APPROVED** [review]

**Pull request merged**